### PR TITLE
[chore](table property) forbid changing enable_single_replica_compaction property for mow table #34836

### PR DESCRIPTION
**Pull request by xiaokang**
cherry pick from #34836

### CODE REVIEW

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

File: regression-test/suites/compaction/test_table_level_compaction_policy.groovy [hunk 1]
```diff
@@ -242,4 +242,28 @@ suite("test_table_level_compaction_policy") {
...
+        CREATE TABLE ${tableName} (
+                `c_custkey` int(11) NOT NULL COMMENT "",
+                `c_name` varchar(26) NOT NULL COMMENT "",
+                `c_address` varchar(41) NOT NULL COMMENT "",
+                `c_city` varchar(11) NOT NULL COMMENT ""
+        )
+        UNIQUE KEY (`c_custkey`)
+        DISTRIBUTED BY HASH(`c_custkey`) BUCKETS 1
+        PROPERTIES (
+                "replication_num" = "1"
```
Comment (csun5285) [inline regression-test/suites/compaction/test_table_level_compaction_policy.groovy:256]: "enable_unique_key_merge_on_write" = "true"

**Comment by xiaokang** [comment]
run buildall

**Pull request merged**